src/vsr/grid_blocks_missing.zig

```diff
@@ -212,12 +213,15 @@ pub const GridBlocksMissing = struct {
     pub fn enqueue_table(
         queue: *GridBlocksMissing,
         table: *RepairTable,
+        table_bitset: *std.DynamicBitSetUnmanaged,
         address: u64,
         checksum: u128,
     ) enum { insert, duplicate } {
         assert(queue.faulty_tables.count < queue.options.tables_max);
         assert(queue.faulty_blocks.count() ==
             queue.enqueued_blocks_single + queue.enqueued_blocks_table);
+        assert(table_bitset.capacity() == constants.lsm_table_data_blocks_max);
+        assert(table_bitset.count() == 0);
 
         var tables = queue.faulty_tables.peek();
         while (tables) |queue_table| : (tables = queue_table.next) {
```

Comment:
In this loop we could also assert that `queue_table.data_box_received != table_bitset`.